echo-pipelinetriggers/src/main/java/com/netflix/spinnaker/echo/pipelinetriggers/orca/PipelineInitiator.java

```diff
@@ -131,6 +131,7 @@ private void triggerPipeline(Pipeline pipeline) {
       try {
         AuthenticatedRequest.propagate(() -> orcaResponse.subscribe(), korkUser).call();
       } catch (Exception e) {
+        registry.counter("orca.error").increment();
```

Comment:
something to consider: maybe add the class of the exception as a tag. Any bit that helps understand what errors are rising on a graph helps